dmutils/apiclient/search.py

```diff
@@ -5,13 +5,29 @@
 
 
 class SearchAPIClient(BaseAPIClient):
+    def __init__(self, *args, **kwargs):
+        self.index_name = kwargs.pop('index_name', 'g-cloud')
```

Comment:
Providing this as a constructor argument doesn't work very well with the way we handle API clients at the moment. Could we just pass it to the functions?